assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -29,17 +29,44 @@ import Data from 'googlesitekit-data';
 import { Select, Option } from '../../../../material-components';
 import ProgressBar from '../../../../components/progress-bar';
 import { STORE_NAME, PROPERTY_CREATE } from '../../datastore/constants';
+import { STORE_NAME as MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import { isValidAccountID } from '../../util';
 import { trackEvent } from '../../../../util';
 const { useSelect, useDispatch } = Data;
 
 export default function PropertySelect() {
-	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
-	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
-	const properties = useSelect( ( select ) => select( STORE_NAME ).getProperties( accountID ) );
-	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
-	const isLoadingAccounts = useSelect( ( select ) => select( STORE_NAME ).isDoingGetAccounts() );
-	const isLoadingProperties = useSelect( ( select ) => select( STORE_NAME ).isDoingGetProperties( accountID ) );
+	const {
+		accountID,
+		propertyID,
+		properties,
+		hasExistingTag,
+		hasGTMTag,
+		isLoadingAccounts,
+		isLoadingProperties,
+	} = useSelect( ( select ) => {
```

Comment:
See above.

Reply:
Split it into two multiline `useSelect` hooks because there is custom logic for selecting related properties.